sequential/lists/solution/lists_exercises.erl

```diff
@@ -101,3 +102,23 @@ run_length_encode([]) ->
 % Any
 list_any(F, List) ->
   lists:foldl(fun(X, Y) -> F(X) or Y end, false, List).
+
+%Anagram
+anagram(List, S) -> 
+    anagram(List, S, []).
+
+anagram([], _S, Acc) -> Acc;
+anagram([H|T], S, Acc) ->
+    LowerH = string:lowercase(H),
+    LowerS = string:lowercase(S),
+    case LowerH == LowerS of
+      true ->
+        anagram(T,S,Acc);
+      false ->
+        case lists:sort(LowerH) == lists:sort(LowerS) of
+          true ->
+            anagram(T, S, [H|Acc]);
+          false->
+            anagram(T,S,Acc)
+    end
+  end.
```

Comment:
You are missing a newline at the end of the file. Your editor should have an option to do this automatically